web/ui/react-app/src/App.tsx

```diff
@@ -69,25 +69,45 @@ const App: FC<AppProps> = ({ consolesLink }) => {
     >
       <Theme />
       <PathPrefixContext.Provider value={basePath}>
-        <Navigation consolesLink={consolesLink} />
-        <Container fluid style={{ paddingTop: 70 }}>
-          <Router basepath={`${basePath}`}>
-            <Redirect from="/" to={`graph`} noThrow />
-            {/*
+        <Router basename={basePath}>
+          <Navigation consolesLink={consolesLink} />
+          <Container fluid style={{ paddingTop: 70 }}>
+            <Switch>
+              <Redirect exact from="/" to={`graph`} />
+              {/*
               NOTE: Any route added here needs to also be added to the list of
```

Comment:
maybe this comment is not accurate anymore ?

Reply:
It's still applicable, as the backend needs to know to serve the React app when pointing the browser at such a path.